assets/export/docx/7x10/style_structure_reference.md

```diff
@@ -0,0 +1,164 @@
+      * Image Caption basedOn: Caption 
+    * header basedOn: FontFamilyDefaultTitleandStructure 
+      * Header: PageNumber basedOn: Header 
+  * Font Family: Default Case Content Body  
+    * Body Text basedOn: FontFamilyDefaultCaseContentBody link BodyTextChar
+      * Body Text First Indent basedOn: BodyText link BodyTextFirstIndentChar
+      * First Paragraph basedOn: BodyText 
+      * Compact basedOn: BodyText 
+      * Case Body basedOn: BodyText 
+        * Author basedOn: CaseBody 
+        * Date basedOn: CaseBody 
+        * Abstract basedOn: CaseBody 
+        * Figure basedOn: CaseBody 
+          * Captioned Figure basedOn: Figure 
+      * Credits basedOn: BodyText 
+      * Quote basedOn: BodyText 
+        * Block Text basedOn: Quote 
+      * Definition basedOn: BodyText 
+        * Definition Term basedOn: Definition 
+        * Image Centered Large basedOn: Definition 
+        * Image Left Medium basedOn: Definition 
+        * Image Right Medium basedOn: Definition 
+        * Image Centered Medium basedOn: Definition 
+      * footer basedOn: BodyText 
+    * footnote text basedOn: FontFamilyDefaultCaseContentBody 
+      * Footnote Labeled Link basedOn: FootnoteText 
+        * Footnote Labeled Case basedOn: FootnoteLabeledLink 
+  * Font Family: Default Source Code  
+    * Source Code basedOn: FontFamilyDefaultSourceCode link VerbatimChar
+  * Normal  
+  * Front Matter End  
+  * Head Separator  
+  * Chapter Spacer  
+  * Head Field Separator  
+  * Head End  
+  * Node Start  
+  * Node End  
+* Character
+  * Default Paragraph Font  
+    * Body Text Char basedOn: DefaultParagraphFont link BodyText
+      * Body Text First Indent Char basedOn: BodyTextChar link BodyTextFirstIndent
+      * Case Footnote Reference basedOn: BodyTextChar 
+    * Unresolved Mention basedOn: DefaultParagraphFont 
+    * H Struct 01 Title Char basedOn: DefaultParagraphFont link HStruct01Title
+    * FollowedHyperlink basedOn: DefaultParagraphFont 
+    * Caption Char basedOn: DefaultParagraphFont link Caption
+  * Verbatim Char  link SourceCode
+  * Elision  
+    * Replacement Text basedOn: Elision 
+  * Highlighted Text  
+  * Hyperlink  
+  * AlertTok  
+  * AnnotationTok  
+  * AttributeTok  
+  * BaseNTok  
+  * BuiltInTok  
+  * CharTok  
+  * CommentTok  
+  * CommentVarTok  
+  * ControlFlowTok  
+  * DataTypeTok  
+  * DocumentationTok  
+  * ErrorTok  
+  * ExtensionTok  
+  * FloatTok  
+  * FunctionTok  
+  * ImportTok  
+  * InformationTok  
+  * KeywordTok  
+  * NormalTok  
+  * OperatorTok  
+  * OtherTok  
+  * PreprocessorTok  
+  * RegionMarkerTok  
+  * SpecialCharTok  
+  * SpecialSTringTok  
+  * StringTok  
+  * VariableTok  
+  * VerbatimStringTok  
+  * WarningTok  
+* Table
+  * Table  
+  * Normal Table  
+    * Plain Table 4 basedOn: TableNormal 
+    * Table Grid basedOn: TableNormal 
+* Numbering
+  * No List  
+  * Current List1  
+  * Current List2  
+  * Current List3  
+  * Current List4  
```

Comment:
Do you know what the story is with `Current List1-4`? Is that just a Word way to have 4 lists around?

Reply:
All of these styles with multiple numbers (Body Text 2, Body Text 3, etc) are built-in word styles. While it's probably not necessary to define everything, I'm worried about defaults slipping through and breaking the document's look and feel.

Reply:
Ah, cool, thx